### ISSUE TITLE
NVDA not reading the first shown/highlighted result when typing in "search" box of Win Vista in start menu.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#715]
**Reported by H.D. on 2010-06-18 13:41**
When typing in the "search" box of Windows Vista, the NVDA does not read/notify the content of the first shown item/highlighted text as appears in the search box. The NVDA latest snapshots too have same problem.

**Comment by leonardder** [comment]
Given @JosephSL wasn't able to reproduce this and Vista is discontinued, I suggest closing this as a works for me.

**Comment by derekriemer** [comment]
I've used this feature on vista before on my Grandma's computer. Worksfome
